src/Uno.Toolkit.UI/Controls/LoadingView/LoadingView.cs

```diff
@@ -137,5 +137,24 @@ private void UpdateVisualState()
 
 			VisualStateManager.GoToState(this, loadingState, IsLoaded);
 		}
+
+		#region DependencyProperty: Source
+
+		public static DependencyProperty SplashScreenProperty { get; } = DependencyProperty.Register(
```

Comment:
I think I'm not understanding something, how come this needs to be a property on LoadingView and not directly on ExtendedSplashScreen? It looks like ExtendedSplashScreen is hiding this property behind its own SplashScreen property.

Was this initially here when SplashScreenPresenter was supposed to be a LoadingView attached property? I'm not seeing this attached property anymore.

Reply:
Splash screen property has been removed